<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMeasurementTypeWidget</class>
 <widget class="QWidget" name="AddMeasurementTypeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Material* :</string>
   </property>
  </widget>
  <widget class="TreeBox" name="materialComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>40</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name* :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="commentsLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pictureLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeBox</class>
   <extends>QComboBox</extends>
   <header>CustomWidgets/treebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
